fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/MixinAbstractFurnaceBlockEntity.java

```diff
@@ -34,4 +35,26 @@
 	private static void fuelTimeMapHook(CallbackInfoReturnable<Map<Item, Integer>> info) {
 		FuelRegistryImpl.INSTANCE.apply(info.getReturnValue());
 	}
+
+	@Redirect(
+			method = "canUseAsFuel",
+			at = @At(
+				value = "INVOKE",
+				target = "Lnet/minecraft/block/entity/AbstractFurnaceBlockEntity;createFuelTimeMap()Ljava/util/Map;"
+			)
+	)
```

Comment:
I personally dislike it but modmuss wants the mixin annotations to be one line

Reply:
Aight